### PR TITLE
Sync repo with previous PR

### LINKED ISSUE #2: Adjust test duration, interval
Individual tests should run for 10 minutes, metrics should be sampled every 10s.

### PR DESCRIPTION
**Pull request by hariso**
I had some changes in https://github.com/ConduitIO/conduit/pull/520 which I didn't move to this repo. This PR closes that gap.

Also fixes #2 .

### CODE REVIEW

File: run-conduit-workload.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+WORKLOAD=$2
+
+printf "\n-- Running %s with %s\n" "$WORKLOAD" "$CONDUIT_IMAGE"
+
+docker stop conduit-perf-test || true
+
+docker run --rm --name conduit-perf-test --memory 1g --cpus=2 -v "$(pwd)/plugins":/plugins -p 8080:8080 -d "$CONDUIT_IMAGE"
+sleep 1
+
+./run-workload.sh "http://localhost:8080" "$WORKLOAD"
```
Comment (lovromazgon) [inline run-conduit-workload.sh:12]: Is it worth having a separate file for this?

File: workloads/large-messages-low-rate.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ echo "Creating a NoOp destination..."
 DEST_CONN_REQ=$(
 jq -n  --arg pipeline_id "$PIPELINE_ID" '{
      "type": "TYPE_DESTINATION",
-     "plugin": "builtin:noopdest",
+     "plugin": "/plugins/conduit-connector-noop-dest",
```
Comment (lovromazgon) [inline workloads/large-messages-low-rate.sh:44]: The plugin binary name could be _a bit_ shorter 😄

**Review by lovromazgon — APPROVED** [review]
Just nitpicks 😉

File: run-conduit-workload.sh (hunk 1 above)
Comment (hariso) [reply run-conduit-workload.sh:12]: Code-wise, I'd say not. But I've personally used it a few times as a shortcut (to execute the script, and then run the tool with the default options). Now that I think of it, I can default the Conduit URL to localhost:8080, to make it a bit shorter.

File: workloads/large-messages-low-rate.sh (hunk 2 above)
Comment (hariso) [reply workloads/large-messages-low-rate.sh:44]: Yeah, I initially wanted to have it consistent with other connectors. But given the very limited use for this destination, I can just call it noop-dest.

**Pull request merged**